sycl/source/CMakeLists.txt

```diff
@@ -150,6 +150,7 @@ set(SYCL_SOURCES
     "sampler.cpp"
     "stream.cpp"
     "spirv_ops.cpp"
+    "interop_handle.cpp"
```

Comment:
I suggest keeping the source list in alphabetical order

Reply:
Done